cdap-client/src/main/java/co/cask/cdap/client/ProgramClient.java

```diff
@@ -323,7 +323,8 @@ public void setFlowletInstances(String appId, String flowId, String flowletId, i
    */
   public int getWorkerInstances(String appId, String workerId) throws IOException, NotFoundException,
     UnauthorizedException {
-    URL url = config.resolveURL(String.format("apps/%s/workers/%s/instances", appId, workerId));
+    URL url = VersionMigrationUtils.resolveURL(config, ProgramType.WORKER,
```

Comment:
We can simply use `config.resolveNamespacedURLV3(path)`, since this endpoint definitely available in v3 APIs.